### ISSUE TITLE
macOS: Code does not start with native tabs

### ISSUE THREAD

**Issue by dalisoft** [microsoft/vscode#117220]
- VS Code Version: v1.53.2
- OS Version: macOS Catalina 10.15.7

Steps to Reproduce:

1. Open VSCode
2. Try use

Does this issue occur when all extensions are disabled?: Yes

https://user-images.githubusercontent.com/3511344/108695512-309f0d80-7522-11eb-9124-972101cd0712.mov

It's not ad, due of GitHub file size limit i needed to use third-party converter

**Comment by dalisoft** [comment]
This caused by by "Always open in tab" in **Settings** -> **Dock**

Related issue #117175

@bpasero It's why not works. Bug in new Electron 11. All bugs which i found appeared after VSCode ARM support or Electron 11. Until it worked perfectly

**Comment by bpasero** [comment]
@dalisoft to clarify, you have enabled this setting?

![image](https://user-images.githubusercontent.com/900690/108715553-52e15d00-751b-11eb-8308-23793a2a4df6.png)

Can you run `code-insiders --verbose` from the command line to see if any output is generated?

**Comment by dalisoft** [comment]
@bpasero Yes, after enabling/setting to always crash/auto closing happens

**Comment by bpasero** [comment]
@dalisoft Can you run `code-insiders --verbose` from the command line to see if any output is generated?

**Comment by dalisoft** [comment]
## Code

```bash
❯ code --verbose
(electron) Sending uncompressed crash reports is deprecated and will be removed in a future version of Electron. Set { compress: true } to opt-in to the new behavior. Crash reports will be uploaded gzipped, which most crash reporting servers support.
[66422:0222/194936.902238:WARNING:dns_config_service_posix.cc(185)] dns_config has unhandled options!
[main 2021-02-22T14:49:37.009Z] Starting VS Code
[main 2021-02-22T14:49:37.010Z] from: /Applications/Visual Studio Code.app/Contents/Resources/app
[main 2021-02-22T14:49:37.010Z] args: {
  _: [],
  diff: false,
  add: false,
  goto: false,
  'new-window': false,
  'reuse-window': false,
  wait: false,
  help: false,
  'list-extensions': false,
  'show-versions': false,
  version: false,
  verbose: true,
  status: false,
  'prof-startup': false,
  'prof-v8-extensions': false,
  'disable-extensions': false,
  'disable-gpu': false,
  telemetry: false,
  debugRenderer: false,
  logExtensionHostCommunication: false,
  'skip-release-notes': false,
  'disable-telemetry': false,
  'disable-updates': false,
  'disable-crash-reporter': false,
  'skip-add-to-recently-opened': false,
  'unity-launch': false,
  'open-url': false,
  'file-write': false,
  'file-chmod': false,
  'driver-verbose': false,
  force: false,
  'do-not-sync': false,
  trace: false,
  'force-user-env': false,
  'force-disable-user-env': false,
  'open-devtools': false,
  __sandbox: false,
  'no-proxy-server': false,
  nolazy: false,
  'force-renderer-accessibility': false,
  'ignore-certificate-errors': false,
  'allow-insecure-localhost': false,
  logsPath: '/Users/dalisoft/Library/Application Support/Code/logs/20210222T194936'
}
[main 2021-02-22T14:49:37.017Z] Resolving machine identifier...
[main 2021-02-22T14:49:37.018Z] Resolved machine identifier: 6a3cf7b0ca7fed5ab37c6479feaa0e062f0b729f22f6e48d85857561a02e792b
[main 2021-02-22T14:49:37.018Z] Main->SharedProcess#connect
[main 2021-02-22T14:49:37.042Z] lifecycle (main): phase changed (value: 2)
[main 2021-02-22T14:49:37.043Z] windowsManager#open
[main 2021-02-22T14:49:37.044Z] windowsManager#open pathsToOpen [
  {
    workspace: { id: '63bba4daa2a07fb24856137a2a30e5b8', uri: [v] },
    exists: true
  },
  {
    workspace: { id: 'a1083744b921f2e6f101f45580037c3b', uri: [v] },
    exists: true
  },
  {
    workspace: { id: '0ccf9878748e54f704ebf033b0fd1fa2', uri: [v] },
    exists: true
  },
  {
    workspace: { id: '0d2bac2551a439f9feb9a52cdce1bb79', uri: [v] },
    exists: true
  },
  {
    workspace: { id: 'c37d2c644b89bb3d40403d82b71aabd7', uri: [v] },
    exists: true
  },
  {
    workspace: { id: '669b26eb933da0b8d682a19193a1cc86', uri: [v] },
    exists: true
  },
  {
    workspace: { id: '563cfefd43dc7bfd7a55f4534287ff88', uri: [v] },
    exists: true
  },
  {
    workspace: { id: '4a00687eaa239fca33ed59a0e021523c', uri: [v] },
    exists: true
  },
  {
    workspace: { id: '58695c13fa3383aab3a2c25f0ca7b3e0', uri: [v] },
    exists: true
  },
  {
    backupPath: '/Users/dalisoft/Library/Application Support/Code/Backups/1614005360679',
    remoteAuthority: undefined
  },
  {
    backupPath: '/Users/dalisoft/Library/Application Support/Code/Backups/1614005360679',
    remoteAuthority: undefined
  }
]
[main 2021-02-22T14:49:37.051Z] window#validateWindowState: validating window state on 1 display(s) { mode: 1, x: 0, y: 23, width: 2560, height: 1353 }
[main 2021-02-22T14:49:37.051Z] window#validateWindowState: 1 monitor working area { x: 0, y: 23, width: 2560, height: 1353 }
[main 2021-02-22T14:49:37.051Z] window#ctor: using window state { mode: 1, x: 0, y: 23, width: 2560, height: 1353 }
[66426:0222/194937.086007:WARNING:dns_config_service_posix.cc(185)] dns_config has unhandled options!
[main 2021-02-22T14:49:37.109Z] window#validateWindowState: validating window state on 1 display(s) { mode: 1, x: 0, y: 23, width: 2560, height: 1353 }
[main 2021-02-22T14:49:37.109Z] window#validateWindowState: 1 monitor working area { x: 0, y: 23, width: 2560, height: 1353 }
[main 2021-02-22T14:49:37.109Z] window#ctor: using window state { mode: 1, x: 0, y: 23, width: 2560, height: 1353 }
[main 2021-02-22T14:49:37.145Z] window#validateWindowState: validating window state on 1 display(s) { mode: 3, display: 69967544, width: 2560, height: 1353, x: 0, y: 23 }
[main 2021-02-22T14:49:37.145Z] window#validateWindowState: 1 monitor working area { x: 0, y: 23, width: 2560, height: 1353 }
[main 2021-02-22T14:49:37.146Z] window#ctor: using window state { mode: 3, display: 69967544, width: 2560, height: 1353, x: 0, y: 23 }
[main 2021-02-22T14:49:37.264Z] window#validateWindowState: validating window state on 1 display(s) { mode: 1, display: 69967544, width: 2560, height: 1353, x: 0, y: 23 }
[main 2021-02-22T14:49:37.264Z] window#validateWindowState: 1 monitor working area { x: 0, y: 23, width: 2560, height: 1353 }
[main 2021-02-22T14:49:37.264Z] window#ctor: using window state { mode: 1, display: 69967544, width: 2560, height: 1353, x: 0, y: 23 }
[main 2021-02-22T14:49:37.294Z] window#validateWindowState: validating window state on 1 display(s) { mode: 1, x: 0, y: 23, width: 2560, height: 1353 }
[main 2021-02-22T14:49:37.294Z] window#validateWindowState: 1 monitor working area { x: 0, y: 23, width: 2560, height: 1353 }
[main 2021-02-22T14:49:37.294Z] window#ctor: using window state { mode: 1, x: 0, y: 23, width: 2560, height: 1353 }
2021-02-22 19:49:37.303 Electron[66422:1103275] *** Assertion failure in -[NSWindowStackController _enterWindow:intoFullScreenWithWindow:], /AppleInternal/BuildRoot/Library/Caches/com.apple.xbs/Sources/AppKit/AppKit-1894.60.100/WindowTabs/NSWindowStackController.m:999
[0222/194937.312789:WARNING:process_memory_mac.cc(93)] mach_vm_read(0x7ffee79f5000, 0x2000): (os/kern) invalid address (1)
[0222/194937.423040:WARNING:crash_report_exception_handler.cc(240)] UniversalExceptionRaise: (os/kern) failure (5)
[66431:0222/194937.427509:ERROR:child_thread_impl.cc(222)] Invalid PlatformChannel receive right
~ 
❯ 
```

## Code Insiders
```bash
❯ code-insiders --verbose
[66683:0222/195057.245434:WARNING:dns_config_service_posix.cc(185)] dns_config has unhandled options!
[main 2021-02-22T14:50:57.359Z] Starting VS Code
[main 2021-02-22T14:50:57.361Z] from: /Applications/Visual Studio Code - Insiders.app/Contents/Resources/app
[main 2021-02-22T14:50:57.361Z] args: {
  _: [],
  diff: false,
  add: false,
  goto: false,
  'new-window': false,
  'reuse-window': false,
  wait: false,
  help: false,
  'list-extensions': false,
  'show-versions': false,
  version: false,
  verbose: true,
  status: false,
  'prof-startup': false,
  'prof-v8-extensions': false,
  'disable-extensions': false,
  'disable-gpu': false,
  telemetry: false,
  debugRenderer: false,
  logExtensionHostCommunication: false,
  'skip-release-notes': false,
  'disable-telemetry': false,
  'disable-updates': false,
  'disable-keytar': false,
  'disable-crash-reporter': false,
  'crash-reporter-id': 'b5d9b0ca-d034-47d2-925a-e51ee7547cdb',
  'skip-add-to-recently-opened': false,
  'unity-launch': false,
  'open-url': false,
  'file-write': false,
  'file-chmod': false,
  'driver-verbose': false,
  force: false,
  'do-not-sync': false,
  trace: false,
  'force-user-env': false,
  'force-disable-user-env': false,
  'open-devtools': false,
  __sandbox: false,
  'no-proxy-server': false,
  nolazy: false,
  'force-renderer-accessibility': false,
  'ignore-certificate-errors': false,
  'allow-insecure-localhost': false,
  logsPath: '/Users/dalisoft/Library/Application Support/Code - Insiders/logs/20210222T195057'
}
[main 2021-02-22T14:50:57.368Z] Resolving machine identifier...
[main 2021-02-22T14:50:57.368Z] Resolved machine identifier: 6a3cf7b0ca7fed5ab37c6479feaa0e062f0b729f22f6e48d85857561a02e792b
[main 2021-02-22T14:50:57.369Z] Main->SharedProcess#connect
[main 2021-02-22T14:50:57.389Z] StorageMainService: creating global storage
[main 2021-02-22T14:50:57.389Z] lifecycle (main): phase changed (value: 2)
[main 2021-02-22T14:50:57.389Z] windowsManager#open
[main 2021-02-22T14:50:57.390Z] windowsManager#open pathsToOpen [
  {
    workspace: { id: '3616f9c71988d4ad5dd50fb21055f3cc', uri: [w] },
    exists: true
  },
  {
    workspace: { id: '669b26eb933da0b8d682a19193a1cc86', uri: [w] },
    exists: true
  },
  {
    workspace: { id: '669b26eb933da0b8d682a19193a1cc86', uri: [w] },
    exists: true
  }
]
[main 2021-02-22T14:50:57.397Z] window#validateWindowState: validating window state on 1 display(s) {
  mode: 3,
  display: 69967544,
  width: 1285,
  height: 1314,
  x: 614,
  y: 50
}
[main 2021-02-22T14:50:57.398Z] window#validateWindowState: 1 monitor working area { x: 0, y: 23, width: 2560, height: 1353 }
[main 2021-02-22T14:50:57.398Z] window#ctor: using window state {
  mode: 3,
  display: 69967544,
  width: 1285,
  height: 1314,
  x: 614,
  y: 50
}
[66688:0222/195057.429410:WARNING:dns_config_service_posix.cc(185)] dns_config has unhandled options!
[main 2021-02-22T14:50:57.514Z] window#validateWindowState: validating window state on 1 display(s) { mode: 1, x: 0, y: 23, width: 2560, height: 1353 }
[main 2021-02-22T14:50:57.514Z] window#validateWindowState: 1 monitor working area { x: 0, y: 23, width: 2560, height: 1353 }
[main 2021-02-22T14:50:57.514Z] window#ctor: using window state { mode: 1, x: 0, y: 23, width: 2560, height: 1353 }
[main 2021-02-22T14:50:57.556Z] windowsManager#open used window count 2 (workspacesToOpen: 0, foldersToOpen: 3, emptyToRestore: 0, emptyToOpen: 0)
[main 2021-02-22T14:50:57.557Z] lifecycle (main): phase changed (value: 3)
[main 2021-02-22T14:50:57.557Z] update#ctor - updates are disabled by user preference
[main 2021-02-22T14:50:57.558Z] resolveShellEnv(): skipped (VSCODE_CLI is set)
(node:66683) electron: The default of contextIsolation is deprecated and will be changing from false to true in a future release of Electron.  See https://github.com/electron/electron/issues/23506 for more information
(node:66683) electron: The default of contextIsolation is deprecated and will be changing from false to true in a future release of Electron.  See https://github.com/electron/electron/issues/23506 for more information
[main 2021-02-22T14:50:57.833Z] resolveShellEnv(): skipped (VSCODE_CLI is set)
[main 2021-02-22T14:50:57.835Z] resolveShellEnv(): skipped (VSCODE_CLI is set)
(node:66690) Electron: Loading non-context-aware native module in renderer: '/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/node_modules.asar.unpacked/vscode-sqlite3/build/Release/sqlite.node'. This is deprecated, see https://github.com/electron/electron/issues/18397.
(node:66689) Electron: Loading non-context-aware native module in renderer: '/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/node_modules.asar.unpacked/vscode-sqlite3/build/Release/sqlite.node'. This is deprecated, see https://github.com/electron/electron/issues/18397.
[66683:0222/195058.117877:INFO:CONSOLE(616)] "%cTRACE color: #888 Error while resolving configuration file 'file:///Users/dalisoft/Desktop/HTML/.vscode/settings.json': Unable to read file '/Users/dalisoft/Desktop/HTML/.vscode/settings.json' (Error: Unable to resolve non-existing file '/Users/dalisoft/Desktop/HTML/.vscode/settings.json')", source: file:///Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (616)
[66683:0222/195058.118429:INFO:CONSOLE(616)] "%cTRACE color: #888 Error while resolving configuration file 'file:///Users/dalisoft/Desktop/HTML/.vscode/tasks.json': Unable to read file '/Users/dalisoft/Desktop/HTML/.vscode/tasks.json' (Error: Unable to resolve non-existing file '/Users/dalisoft/Desktop/HTML/.vscode/tasks.json')", source: file:///Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (616)
[66683:0222/195058.118472:INFO:CONSOLE(616)] "%cTRACE color: #888 Error while resolving configuration file 'file:///Users/dalisoft/Desktop/HTML/.vscode/launch.json': Unable to read file '/Users/dalisoft/Desktop/HTML/.vscode/launch.json' (Error: Unable to resolve non-existing file '/Users/dalisoft/Desktop/HTML/.vscode/launch.json')", source: file:///Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (616)
[66683:0222/195058.118934:INFO:CONSOLE(616)] "%cTRACE color: #888 Error while resolving configuration file 'file:///Users/dalisoft/Desktop/Workdir/ADV/MyScreen/myscreen-frontend/.vscode/settings.json': Unable to read file '/Users/dalisoft/Desktop/Workdir/ADV/MyScreen/myscreen-frontend/.vscode/settings.json' (Error: Unable to resolve non-existing file '/Users/dalisoft/Desktop/Workdir/ADV/MyScreen/myscreen-frontend/.vscode/settings.json')", source: file:///Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (616)
[66683:0222/195058.119218:INFO:CONSOLE(616)] "%cTRACE color: #888 Error while resolving configuration file 'file:///Users/dalisoft/Desktop/Workdir/ADV/MyScreen/myscreen-frontend/.vscode/tasks.json': Unable to read file '/Users/dalisoft/Desktop/Workdir/ADV/MyScreen/myscreen-frontend/.vscode/tasks.json' (Error: Unable to resolve non-existing file '/Users/dalisoft/Desktop/Workdir/ADV/MyScreen/myscreen-frontend/.vscode/tasks.json')", source: file:///Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (616)
[66683:0222/195058.119254:INFO:CONSOLE(616)] "%cTRACE color: #888 Error while resolving configuration file 'file:///Users/dalisoft/Desktop/Workdir/ADV/MyScreen/myscreen-frontend/.vscode/launch.json': Unable to read file '/Users/dalisoft/Desktop/Workdir/ADV/MyScreen/myscreen-frontend/.vscode/launch.json' (Error: Unable to resolve non-existing file '/Users/dalisoft/Desktop/Workdir/ADV/MyScreen/myscreen-frontend/.vscode/launch.json')", source: file:///Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (616)
[66683:0222/195058.127454:INFO:CONSOLE(616)] "%cTRACE color: #888 [lifecycle] starting up (startup kind: 1)", source: file:///Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (616)
[66683:0222/195058.127959:INFO:CONSOLE(616)] "%cTRACE color: #888 lifecycle: phase changed (value: 2)", source: file:///Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (616)
[66683:0222/195058.129894:INFO:CONSOLE(616)] "%cTRACE color: #888 [lifecycle] starting up (startup kind: 1)", source: file:///Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (616)
[66683:0222/195058.129968:INFO:CONSOLE(616)] "%cTRACE color: #888 lifecycle: phase changed (value: 2)", source: file:///Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (616)
[66683:0222/195058.130012:INFO:CONSOLE(616)] "%cTRACE color: #888 Renderer->SharedProcess#connect", source: file:///Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (616)
[66683:0222/195058.130895:INFO:CONSOLE(616)] "%cTRACE color: #888 Renderer->SharedProcess#connect", source: file:///Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (616)
[main 2021-02-22T14:50:58.130Z] SharedProcess: on vscode:createSharedProcessMessageChannel
[main 2021-02-22T14:50:58.130Z] SharedProcess: on vscode:createSharedProcessMessageChannel
[66683:0222/195058.134130:INFO:CONSOLE(616)] "%cTRACE color: #888 Skipping initializing user data in desktop", source: file:///Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (616)
[66683:0222/195058.135026:INFO:CONSOLE(616)] "%cTRACE color: #888 Skipping initializing user data in desktop", source: file:///Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (616)
[66683:0222/195058.249073:INFO:CONSOLE(616)] "%cTRACE color: #888 Layout#layout, height: 1414, width: 2560", source: file:///Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (616)
[66683:0222/195058.260420:INFO:CONSOLE(616)] "%cTRACE color: #888 Layout#layout, height: 1305, width: 2560", source: file:///Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (616)
[66683:0222/195058.290721:INFO:CONSOLE(616)] "%cTRACE color: #888 workbench configuration {"_":[],"verbose":true,"crash-reporter-id":"b5d9b0ca-d034-47d2-925a-e51ee7547cdb","logsPath":"/Users/dalisoft/Library/Application Support/Code - Insiders/logs/20210222T195057","appRoot":"/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app","machineId":"6a3cf7b0ca7fed5ab37c6479feaa0e062f0b729f22f6e48d85857561a02e792b","nodeCachedDataDir":"/Users/dalisoft/Library/Application Support/Code - Insiders/CachedData/192c817fd350bcbf3caecae22a45ec39bae78516","mainPid":66683,"execPath":"/Applications/Visual Studio Code - Insiders.app/Contents/MacOS/Electron","userEnv":{"VSCODE_IPC_HOOK":"/Users/dalisoft/Library/Application Support/Code - Insiders/1.54.0-insider-main.sock","VSCODE_NLS_CONFIG":"{\"locale\":\"ru\",\"availableLanguages\":{},\"_languagePackSupport\":true}"},"isInitialStartup":true,"workspace":{"id":"3616f9c71988d4ad5dd50fb21055f3cc","uri":{"$mid":1,"fsPath":"/Users/dalisoft/Desktop/HTML","external":"file:///Users/dalisoft/Desktop/HTML","path":"/Users/dalisoft/Desktop/HTML","scheme":"file"}},"backupPath":"/Users/dalisoft/Library/Application Support/Code - Insiders/Backups/b1428a66b032066585a4c3e57d3381e3","windowId":1,"sessionId":"window:1","logLevel":0,"fullscreen":true,"colorScheme":{"dark":true,"highContrast":false},"autoDetectHighContrast":true,"perfMarks":[{"name":"code/timeOrigin","startTime":1614005457108},{"name":"code/didStartMain","startTime":1614005457213},{"name":"code/mainAppReady","startTime":1614005457278},{"name":"code/willGenerateNls","startTime":1614005457279},{"name":"code/didGenerateNls","startTime":1614005457279},{"name":"code/willLoadMainBundle","startTime":1614005457293},{"name":"code/fork/willLoadCode","startTime":1614005457305},{"name":"code/registerFilesystem/file","startTime":1614005457342},{"name":"code/didLoadMainBundle","startTime":1614005457343},{"name":"code/willOpenNewWindow","startTime":1614005457497}],"partsSplashPath":"/Users/dalisoft/Library/Application Support/Code - Insiders/rapid_render.json","os":{"release":"19.6.0"}}", source: file:///Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (616)
[66683:0222/195058.310362:INFO:CONSOLE(616)] "%cTRACE color: #888 [text file model] load() - enter file:///Users/dalisoft/Desktop/HTML/register-sent.html", source: file:///Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (616)
[66683:0222/195058.312368:INFO:CONSOLE(616)] "%cTRACE color: #888 workbench configuration {"_":[],"verbose":true,"crash-reporter-id":"b5d9b0ca-d034-47d2-925a-e51ee7547cdb","logsPath":"/Users/dalisoft/Library/Application Support/Code - Insiders/logs/20210222T195057","appRoot":"/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app","machineId":"6a3cf7b0ca7fed5ab37c6479feaa0e062f0b729f22f6e48d85857561a02e792b","nodeCachedDataDir":"/Users/dalisoft/Library/Application Support/Code - Insiders/CachedData/192c817fd350bcbf3caecae22a45ec39bae78516","mainPid":66683,"execPath":"/Applications/Visual Studio Code - Insiders.app/Contents/MacOS/Electron","userEnv":{"VSCODE_IPC_HOOK":"/Users/dalisoft/Library/Application Support/Code - Insiders/1.54.0-insider-main.sock","VSCODE_NLS_CONFIG":"{\"locale\":\"ru\",\"availableLanguages\":{},\"_languagePackSupport\":true}"},"isInitialStartup":true,"workspace":{"id":"669b26eb933da0b8d682a19193a1cc86","uri":{"$mid":1,"fsPath":"/Users/dalisoft/Desktop/Workdir/ADV/MyScreen/myscreen-frontend","external":"file:///Users/dalisoft/Desktop/Workdir/ADV/MyScreen/myscreen-frontend","path":"/Users/dalisoft/Desktop/Workdir/ADV/MyScreen/myscreen-frontend","scheme":"file"}},"backupPath":"/Users/dalisoft/Library/Application Support/Code - Insiders/Backups/8c1944520787fcddf1663d282e10fce6","windowId":2,"sessionId":"window:2","logLevel":0,"colorScheme":{"dark":true,"highContrast":false},"autoDetectHighContrast":true,"maximized":true,"perfMarks":[{"name":"code/timeOrigin","startTime":1614005457108},{"name":"code/didStartMain","startTime":1614005457213},{"name":"code/mainAppReady","startTime":1614005457278},{"name":"code/willGenerateNls","startTime":1614005457279},{"name":"code/didGenerateNls","startTime":1614005457279},{"name":"code/willLoadMainBundle","startTime":1614005457293},{"name":"code/fork/willLoadCode","startTime":1614005457305},{"name":"code/registerFilesystem/file","startTime":1614005457342},{"name":"code/didLoadMainBundle","startTime":1614005457343},{"name":"code/willOpenNewWindow","startTime":1614005457497},{"name":"code/willOpenNewWindow","startTime":1614005457552}],"partsSplashPath":"/Users/dalisoft/Library/Application Support/Code - Insiders/rapid_render.json","os":{"release":"19.6.0"}}", source: file:///Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (616)
[66683:0222/195058.386387:INFO:CONSOLE(616)] "%cTRACE color: #888 [text file model] load() - enter file:///Users/dalisoft/Desktop/Workdir/ADV/MyScreen/myscreen-frontend/src/helpers/try-catchy.ts", source: file:///Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (616)
[main 2021-02-22T14:50:58.388Z] menubarService#updateMenubar 1
[66683:0222/195058.439701:INFO:CONSOLE(616)] "%cTRACE color: #888 [text file model] loadFromFile() file:///Users/dalisoft/Desktop/HTML/register-sent.html", source: file:///Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (616)
[main 2021-02-22T14:50:58.442Z] menubarService#updateMenubar 2
[66683:0222/195058.510260:INFO:CONSOLE(616)] "%cTRACE color: #888 [text file model] loadFromFile() file:///Users/dalisoft/Desktop/Workdir/ADV/MyScreen/myscreen-frontend/src/helpers/try-catchy.ts", source: file:///Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (616)
[66683:0222/195058.570968:INFO:CONSOLE(616)] "%cTRACE color: #888 [text file model] loadFromContent() - enter file:///Users/dalisoft/Desktop/HTML/register-sent.html", source: file:///Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (616)
[66683:0222/195058.573624:INFO:CONSOLE(616)] "%cTRACE color: #888 [text file model] doCreateTextModel() file:///Users/dalisoft/Desktop/HTML/register-sent.html", source: file:///Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (616)
[66683:0222/195058.595953:INFO:CONSOLE(616)] "%cTRACE color: #888 [text file model] loadFromContent() - enter file:///Users/dalisoft/Desktop/Workdir/ADV/MyScreen/myscreen-frontend/src/helpers/try-catchy.ts", source: file:///Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (616)
[66683:0222/195058.598000:INFO:CONSOLE(616)] "%cTRACE color: #888 [text file model] doCreateTextModel() file:///Users/dalisoft/Desktop/Workdir/ADV/MyScreen/myscreen-frontend/src/helpers/try-catchy.ts", source: file:///Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (616)
[66683:0222/195058.697384:INFO:CONSOLE(616)] "%cTRACE color: #888 lifecycle: phase changed (value: 3)", source: file:///Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (616)
[66683:0222/195058.722794:INFO:CONSOLE(616)] "%cTRACE color: #888 lifecycle: phase changed (value: 3)", source: file:///Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (616)
[66683:0222/195058.827172:INFO:CONSOLE(1848)] "[matklad.rust-analyzer]: 'configuration.property' must be an object", source: file:///Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (1848)
[66683:0222/195058.827608:INFO:CONSOLE(1848)] "[matklad.rust-analyzer]: 'configuration.property' must be an object", source: file:///Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (1848)
[66683:0222/195058.968629:INFO:CONSOLE(1848)] "[matklad.rust-analyzer]: 'configuration.property' must be an object", source: file:///Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (1848)
[66683:0222/195058.968698:INFO:CONSOLE(1848)] "[matklad.rust-analyzer]: 'configuration.property' must be an object", source: file:///Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (1848)
[66683:0222/195058.978826:INFO:CONSOLE(73)] "Overwriting extension <<.editorconfig>> to now point to mime <<text/x-editorconfig>>", source: file:///Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (73)
[66683:0222/195058.981072:INFO:CONSOLE(73)] "Overwriting extension <<.conf>> to now point to mime <<text/x-apacheconf>>", source: file:///Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (73)
[66683:0222/195059.035295:INFO:CONSOLE(616)] "%cTRACE color: #888 Settings Sync: Initializing accounts", source: file:///Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (616)
[66683:0222/195059.035691:INFO:CONSOLE(616)] "%cTRACE color: #888 Settings Sync: Getting accounts for github", source: file:///Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (616)
[66683:0222/195059.037039:INFO:CONSOLE(73)] "Overwriting extension <<.editorconfig>> to now point to mime <<text/x-editorconfig>>", source: file:///Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (73)
[66683:0222/195059.039835:INFO:CONSOLE(73)] "Overwriting extension <<.conf>> to now point to mime <<text/x-apacheconf>>", source: file:///Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (73)
[66683:0222/195059.106490:INFO:CONSOLE(616)] "%cDEBUG background: #eee; color: #888 [File Watcher (chokidar)] Start watching with chokidar: /Users/dalisoft/Desktop/HTML, excludes: **/.cache/**,**/.git/objects/**,**/.git/subtree-cache/**,**/.hg/store/**,**/.idea/**,**/.webpack/**,**/build,**/dist,**/node_modules/**,**/tmp,**/*.asar, usePolling: false", source: file:///Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (616)
[66683:0222/195059.154808:INFO:CONSOLE(1861)] "Overwriting grammar scope name to file mapping for scope text.log.
Old grammar file: file:///Applications/Visual%20Studio%20Code%20-%20Insiders.app/Contents/Resources/app/extensions/log/syntaxes/log.tmLanguage.json.
New grammar file: file:///Users/dalisoft/.vscode-insiders/extensions/emilast.logfilehighlighter-2.11.0/syntaxes/log.tmLanguage", source: file:///Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (1861)
[66683:0222/195059.154867:INFO:CONSOLE(1861)] "Overwriting grammar scope name to file mapping for scope source.java.
Old grammar file: file:///Applications/Visual%20Studio%20Code%20-%20Insiders.app/Contents/Resources/app/extensions/java/syntaxes/java.tmLanguage.json.
New grammar file: file:///Users/dalisoft/.vscode-insiders/extensions/redhat.java-0.75.0/syntaxes/java.tmLanguage.json", source: file:///Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (1861)
[66683:0222/195059.155159:INFO:CONSOLE(1861)] "Overwriting grammar scope name to file mapping for scope source.yaml.
Old grammar file: file:///Applications/Visual%20Studio%20Code%20-%20Insiders.app/Contents/Resources/app/extensions/yaml/syntaxes/yaml.tmLanguage.json.
New grammar file: file:///Users/dalisoft/.vscode-insiders/extensions/redhat.vscode-yaml-0.15.0/syntaxes/yaml.tmLanguage.json", source: file:///Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (1861)
[66683:0222/195059.190896:INFO:CONSOLE(616)] "%cTRACE color: #888 Settings Sync: Initializing accounts", source: file:///Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (616)
[66683:0222/195059.191288:INFO:CONSOLE(616)] "%cTRACE color: #888 Settings Sync: Getting accounts for github", source: file:///Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (616)
[66683:0222/195059.269254:INFO:CONSOLE(616)] "%cTRACE color: #888 Error while resolving configuration file 'vscode-userdata:/Users/dalisoft/Library/Application%20Support/Code%20-%20Insiders/User/tasks.json': Unable to read file 'vscode-userdata:/Users/dalisoft/Library/Application Support/Code - Insiders/User/tasks.json' (Error: Unable to resolve non-existing file 'vscode-userdata:/Users/dalisoft/Library/Application Support/Code - Insiders/User/tasks.json')", source: file:///Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (616)
[66683:0222/195059.292067:INFO:CONSOLE(616)] "%cDEBUG background: #eee; color: #888 [File Watcher (chokidar)] Start watching with chokidar: /Users/dalisoft/Desktop/Workdir/ADV/MyScreen/myscreen-frontend, excludes: **/.cache/**,**/.git/objects/**,**/.git/subtree-cache/**,**/.hg/store/**,**/.idea/**,**/.webpack/**,**/build,**/dist,**/node_modules/**,**/tmp,**/*.asar, usePolling: false", source: file:///Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (616)
[main 2021-02-22T14:50:59.346Z] menubarService#updateMenubar 1
[66683:0222/195059.403211:INFO:CONSOLE(1861)] "Overwriting grammar scope name to file mapping for scope text.log.
Old grammar file: file:///Applications/Visual%20Studio%20Code%20-%20Insiders.app/Contents/Resources/app/extensions/log/syntaxes/log.tmLanguage.json.
New grammar file: file:///Users/dalisoft/.vscode-insiders/extensions/emilast.logfilehighlighter-2.11.0/syntaxes/log.tmLanguage", source: file:///Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (1861)
[66683:0222/195059.403295:INFO:CONSOLE(1861)] "Overwriting grammar scope name to file mapping for scope source.java.
Old grammar file: file:///Applications/Visual%20Studio%20Code%20-%20Insiders.app/Contents/Resources/app/extensions/java/syntaxes/java.tmLanguage.json.
New grammar file: file:///Users/dalisoft/.vscode-insiders/extensions/redhat.java-0.75.0/syntaxes/java.tmLanguage.json", source: file:///Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (1861)
[66683:0222/195059.403338:INFO:CONSOLE(1861)] "Overwriting grammar scope name to file mapping for scope source.yaml.
Old grammar file: file:///Applications/Visual%20Studio%20Code%20-%20Insiders.app/Contents/Resources/app/extensions/yaml/syntaxes/yaml.tmLanguage.json.
New grammar file: file:///Users/dalisoft/.vscode-insiders/extensions/redhat.vscode-yaml-0.15.0/syntaxes/yaml.tmLanguage.json", source: file:///Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (1861)
[66683:0222/195059.507519:INFO:CONSOLE(616)] "%cTRACE color: #888 SearchService#search {"_reason":"checkExists","folderQueries":[{"folder":{"$mid":1,"fsPath":"/Users/dalisoft/Desktop/HTML","external":"file:///Users/dalisoft/Desktop/HTML","path":"/Users/dalisoft/Desktop/HTML","scheme":"file"},"excludePattern":{"**/.git":true,"**/.svn":true,"**/.hg":true,"**/CVS":true,"**/.DS_Store":true,"**/node_modules":true,"**/bower_components":true,"**/*.code-search":true,"**/.cache":true,"**/.out":true,"**/build":true,"**/dist":true,"**/tmp":true,"**/bower_packages":true},"fileEncoding":"utf8","disregardIgnoreFiles":false,"disregardGlobalIgnoreFiles":true,"ignoreSymlinks":false}],"usingSearchPaths":false,"includePattern":{"**/.dprintrc.json":true},"type":1,"exists":true}", source: file:///Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (616)
[66683:0222/195059.507889:INFO:CONSOLE(616)] "%cTRACE color: #888 SearchService#search {"_reason":"checkExists","folderQueries":[{"folder":{"$mid":1,"fsPath":"/Users/dalisoft/Desktop/HTML","external":"file:///Users/dalisoft/Desktop/HTML","path":"/Users/dalisoft/Desktop/HTML","scheme":"file"},"excludePattern":{"**/.git":true,"**/.svn":true,"**/.hg":true,"**/CVS":true,"**/.DS_Store":true,"**/node_modules":true,"**/bower_components":true,"**/*.code-search":true,"**/.cache":true,"**/.out":true,"**/build":true,"**/dist":true,"**/tmp":true,"**/bower_packages":true},"fileEncoding":"utf8","disregardIgnoreFiles":false,"disregardGlobalIgnoreFiles":true,"ignoreSymlinks":false}],"usingSearchPaths":false,"includePattern":{"**/*.go":true},"type":1,"exists":true}", source: file:///Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (616)
[66683:0222/195059.508187:INFO:CONSOLE(616)] "%cTRACE color: #888 SearchService#search {"_reason":"checkExists","folderQueries":[{"folder":{"$mid":1,"fsPath":"/Users/dalisoft/Desktop/HTML","external":"file:///Users/dalisoft/Desktop/HTML","path":"/Users/dalisoft/Desktop/HTML","scheme":"file"},"excludePattern":{"**/.git":true,"**/.svn":true,"**/.hg":true,"**/CVS":true,"**/.DS_Store":true,"**/node_modules":true,"**/bower_components":true,"**/*.code-search":true,"**/.cache":true,"**/.out":true,"**/build":true,"**/dist":true,"**/tmp":true,"**/bower_packages":true},"fileEncoding":"utf8","disregardIgnoreFiles":false,"disregardGlobalIgnoreFiles":true,"ignoreSymlinks":false}],"usingSearchPaths":false,"includePattern":{"**/Cargo.toml":true},"type":1,"exists":true}", source: file:///Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (616)
[66683:0222/195059.578474:INFO:CONSOLE(616)] "%cTRACE color: #888 [File Watcher (node.js)] [ADDED] /Users/dalisoft/Library/Application Support/Code - Insiders/logs/20210222T195057/exthost1", source: file:///Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (616)
[66683:0222/195059.651247:INFO:CONSOLE(616)] "%cDEBUG background: #eee; color: #888 SearchService#search /Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/node_modules.asar.unpacked/vscode-ripgrep/bin/rg --files --hidden --case-sensitive -g '**/*.go' -g '!**/.git' -g '!**/.svn' -g '!**/.hg' -g '!**/CVS' -g '!**/.DS_Store' -g '!**/node_modules' -g '!**/bower_components' -g '!**/*.code-search' -g '!**/.cache' -g '!**/.out' -g '!**/build' -g '!**/dist' -g '!**/tmp' -g '!**/bower_packages' --no-ignore-parent --follow --quiet --no-config --no-ignore-global
 - cwd: /Users/dalisoft/Desktop/HTML
 - Sibling clauses: {}", source: file:///Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (616)
[66683:0222/195059.653444:INFO:CONSOLE(616)] "%cDEBUG background: #eee; color: #888 SearchService#search /Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/node_modules.asar.unpacked/vscode-ripgrep/bin/rg --files --hidden --case-sensitive -g '**/Cargo.toml' -g '!**/.git' -g '!**/.svn' -g '!**/.hg' -g '!**/CVS' -g '!**/.DS_Store' -g '!**/node_modules' -g '!**/bower_components' -g '!**/*.code-search' -g '!**/.cache' -g '!**/.out' -g '!**/build' -g '!**/dist' -g '!**/tmp' -g '!**/bower_packages' --no-ignore-parent --follow --quiet --no-config --no-ignore-global
 - cwd: /Users/dalisoft/Desktop/HTML
 - Sibling clauses: {}", source: file:///Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (616)
[66683:0222/195059.655042:INFO:CONSOLE(616)] "%cDEBUG background: #eee; color: #888 SearchService#search /Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/node_modules.asar.unpacked/vscode-ripgrep/bin/rg --files --hidden --case-sensitive -g '**/.dprintrc.json' -g '!**/.git' -g '!**/.svn' -g '!**/.hg' -g '!**/CVS' -g '!**/.DS_Store' -g '!**/node_modules' -g '!**/bower_components' -g '!**/*.code-search' -g '!**/.cache' -g '!**/.out' -g '!**/build' -g '!**/dist' -g '!**/tmp' -g '!**/bower_packages' --no-ignore-parent --follow --quiet --no-config --no-ignore-global
 - cwd: /Users/dalisoft/Desktop/HTML
 - Sibling clauses: {}", source: file:///Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (616)
[66683:0222/195059.658926:INFO:CONSOLE(616)] "%cTRACE color: #888 SearchService#search: 147ms", source: file:///Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (616)
[66683:0222/195059.660884:INFO:CONSOLE(616)] "%cTRACE color: #888 SearchService#search: 149ms", source: file:///Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (616)
[66683:0222/195059.665275:INFO:CONSOLE(616)] "%cTRACE color: #888 SearchService#search: 154ms", source: file:///Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (616)
[main 2021-02-22T14:50:59.768Z] menubarService#updateMenubar 1
[66683:0222/195059.780182:INFO:CONSOLE(616)] "%cTRACE color: #888 [File Watcher (node.js)] >> normalized [ADDED] /Users/dalisoft/Library/Application Support/Code - Insiders/logs/20210222T195057/exthost1", source: file:///Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (616)
[66683:0222/195059.783100:INFO:CONSOLE(616)] "%cTRACE color: #888 MainThreadTreeViews#$registerTreeViewDataProvider cSpellRegExpView [object Object]", source: file:///Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (616)
[66683:0222/195059.784566:INFO:CONSOLE(616)] "%cTRACE color: #888 CommandService#executeCommand extension.js-debug.clearAutoAttachVariables", source: file:///Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (616)
[66683:0222/195059.853142:INFO:CONSOLE(616)] "%cTRACE color: #888 Error while resolving configuration file 'vscode-userdata:/Users/dalisoft/Library/Application%20Support/Code%20-%20Insiders/User/tasks.json': Unable to read file 'vscode-userdata:/Users/dalisoft/Library/Application Support/Code - Insiders/User/tasks.json' (Error: Unable to resolve non-existing file 'vscode-userdata:/Users/dalisoft/Library/Application Support/Code - Insiders/User/tasks.json')", source: file:///Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (616)
[66683:0222/195100.006423:INFO:CONSOLE(616)] "%cTRACE color: #888 SearchService#search {"_reason":"checkExists","folderQueries":[{"folder":{"$mid":1,"fsPath":"/Users/dalisoft/Desktop/Workdir/ADV/MyScreen/myscreen-frontend","external":"file:///Users/dalisoft/Desktop/Workdir/ADV/MyScreen/myscreen-frontend","path":"/Users/dalisoft/Desktop/Workdir/ADV/MyScreen/myscreen-frontend","scheme":"file"},"excludePattern":{"**/.git":true,"**/.svn":true,"**/.hg":true,"**/CVS":true,"**/.DS_Store":true,"**/node_modules":true,"**/bower_components":true,"**/*.code-search":true,"**/.cache":true,"**/.out":true,"**/build":true,"**/dist":true,"**/tmp":true,"**/bower_packages":true},"fileEncoding":"utf8","disregardIgnoreFiles":false,"disregardGlobalIgnoreFiles":true,"ignoreSymlinks":false}],"usingSearchPaths":false,"includePattern":{"**/.dprintrc.json":true},"type":1,"exists":true}", source: file:///Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (616)
[66683:0222/195100.007301:INFO:CONSOLE(616)] "%cTRACE color: #888 SearchService#search {"_reason":"checkExists","folderQueries":[{"folder":{"$mid":1,"fsPath":"/Users/dalisoft/Desktop/Workdir/ADV/MyScreen/myscreen-frontend","external":"file:///Users/dalisoft/Desktop/Workdir/ADV/MyScreen/myscreen-frontend","path":"/Users/dalisoft/Desktop/Workdir/ADV/MyScreen/myscreen-frontend","scheme":"file"},"excludePattern":{"**/.git":true,"**/.svn":true,"**/.hg":true,"**/CVS":true,"**/.DS_Store":true,"**/node_modules":true,"**/bower_components":true,"**/*.code-search":true,"**/.cache":true,"**/.out":true,"**/build":true,"**/dist":true,"**/tmp":true,"**/bower_packages":true},"fileEncoding":"utf8","disregardIgnoreFiles":false,"disregardGlobalIgnoreFiles":true,"ignoreSymlinks":false}],"usingSearchPaths":false,"includePattern":{"**/*.go":true},"type":1,"exists":true}", source: file:///Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (616)
[66683:0222/195100.007845:INFO:CONSOLE(616)] "%cTRACE color: #888 SearchService#search {"_reason":"checkExists","folderQueries":[{"folder":{"$mid":1,"fsPath":"/Users/dalisoft/Desktop/Workdir/ADV/MyScreen/myscreen-frontend","external":"file:///Users/dalisoft/Desktop/Workdir/ADV/MyScreen/myscreen-frontend","path":"/Users/dalisoft/Desktop/Workdir/ADV/MyScreen/myscreen-frontend","scheme":"file"},"excludePattern":{"**/.git":true,"**/.svn":true,"**/.hg":true,"**/CVS":true,"**/.DS_Store":true,"**/node_modules":true,"**/bower_components":true,"**/*.code-search":true,"**/.cache":true,"**/.out":true,"**/build":true,"**/dist":true,"**/tmp":true,"**/bower_packages":true},"fileEncoding":"utf8","disregardIgnoreFiles":false,"disregardGlobalIgnoreFiles":true,"ignoreSymlinks":false}],"usingSearchPaths":false,"includePattern":{"**/Cargo.toml":true},"type":1,"exists":true}", source: file:///Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (616)
[66683:0222/195100.022203:INFO:CONSOLE(616)] "%cTRACE color: #888 Settings Sync: Updated accounts for github", source: file:///Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (616)
[66683:0222/195100.038207:INFO:CONSOLE(616)] "%cTRACE color: #888 CommandService#executeCommand setContext", source: file:///Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (616)
[66683:0222/195100.038686:INFO:CONSOLE(616)] "%cTRACE color: #888 CommandService#executeCommand setContext", source: file:///Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (616)
[main 2021-02-22T14:51:00.059Z] menubarService#updateMenubar 2
[66683:0222/195100.071949:INFO:CONSOLE(616)] "%cTRACE color: #888 [File Watcher (node.js)] [ADDED] /Users/dalisoft/Library/Application Support/Code - Insiders/logs/20210222T195057/exthost2", source: file:///Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (616)
[66683:0222/195100.072267:INFO:CONSOLE(616)] "%cTRACE color: #888 MainThreadTreeViews#$registerTreeViewDataProvider jsBrowserBreakpoints [object Object]", source: file:///Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (616)
[66683:0222/195100.072307:INFO:CONSOLE(616)] "%cTRACE color: #888 [File Watcher (node.js)] [ADDED] /Users/dalisoft/Library/Application Support/Code - Insiders/logs/20210222T195057/exthost1", source: file:///Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (616)
2021-02-22 19:51:00.167 Electron[66683:1106754] *** Assertion failure in -[NSWindowStackController _enterWindow:intoFullScreenWithWindow:], /AppleInternal/BuildRoot/Library/Caches/com.apple.xbs/Sources/AppKit/AppKit-1894.60.100/WindowTabs/NSWindowStackController.m:999
[0222/195100.175591:WARNING:process_memory_mac.cc(93)] mach_vm_read(0x7ffee431d000, 0x2000): (os/kern) invalid address (1)
[0222/195100.515001:ERROR:http_transport_mac.mm(283)] Could not connect to the server. (NSURLErrorDomain -1004)
[0222/195102.588340:WARNING:crash_report_exception_handler.cc(240)] UniversalExceptionRaise: (os/kern) failure (5)
~ took 5s 
❯ 
```

**Comment by dalisoft** [comment]
@bpasero Setting to "only Fullscreen" and setting **Restore fullscreen** also does not restores window state

**Comment by dalisoft** [comment]
My config

<details>
<summary>Code (Insiders) Settings</summary>

```json
{
  "sync.gist": "376c3dfc48eae5a95d15cfc87639f6cf",
  "[shellscript]": {
    "editor.defaultFormatter": "foxundermoon.shell-format"
  },
  "prettier.requireConfig": true,
  "editor.defaultFormatter": "esbenp.prettier-vscode",
  "editor.scrollBeyondLastLine": false,
  "editor.scrollBeyondLastColumn": 0,
  "editor.renderControlCharacters": true,
  "editor.renderWhitespace": "all",
  "editor.detectIndentation": true,
  "editor.renderIndentGuides": true,
  "editor.cursorBlinking": "phase",
  "editor.cursorSmoothCaretAnimation": true,
  "editor.formatOnPaste": false,
  "editor.formatOnType": false,
  "editor.formatOnSave": true,
  "editor.linkedEditing": true,
  "editor.codeActionsOnSave": ["source.fixAll", "source.organizeImports"],
  "editor.minimap.enabled": true,
  "editor.minimap.renderCharacters": false,
  "editor.minimap.maxColumn": 80,
  "editor.minimap.showSlider": "always",
  "editor.smoothScrolling": true,
  "editor.quickSuggestions": true,
  "editor.quickSuggestionsDelay": 100,
  "editor.acceptSuggestionOnEnter": "smart",
  "editor.suggestSelection": "first",
  "editor.suggestOnTriggerCharacters": true,
  "editor.suggest.showInlineDetails": true,
  "editor.suggest.localityBonus": true,
  "editor.wordBasedSuggestionsMode": "allDocuments",
  "editor.acceptSuggestionOnCommitCharacter": true,
  "editor.parameterHints.enabled": true,
  "editor.snippetSuggestions": "top",
  "editor.definitionLinkOpensInPeek": true,
  "editor.fontSize": 14,
  "editor.fontFamily": "Monaco, Fira Code",
  "editor.codeLensFontSize": 14,
  "editor.codeLensFontFamily": "Fira Code",
  "editor.tabCompletion": "on",
  "editor.wordWrap": "bounded",
  "editor.tabSize": 2,
  "editor.stickyTabStops": true,
  "editor.codeLens": true,
  "zenMode.restore": true,
  "terminal.integrated.minimumContrastRatio": 4.5,
  "terminal.integrated.fontFamily": "Fira Code",
  "terminal.integrated.fontWeight": "100",
  "terminal.integrated.fontWeightBold": "400",
  "terminal.integrated.fontSize": 12,
  "terminal.integrated.rendererType": "auto",
  "terminal.integrated.experimentalRefreshOnResume": true,
  "terminal.integrated.experimentalUseTitleEvent": true,
  "terminal.integrated.experimentalLinkProvider": true,
  "terminal.explorerKind": "external",
  "timeline.showView": true,
  "breadcrumbs.enabled": true,
  "extensions.ignoreRecommendations": false,
  "extensions.autoUpdate": true,
  "extensions.autoCheckUpdates": true,
  "telemetry.enableCrashReporter": false,
  "telemetry.enableTelemetry": false,
  "code-runner.enableAppInsights": false,
  "update.mode": "none",
  "update.showReleaseNotes": false,
  "workbench.quickOpen.preserveInput": true,
  "workbench.commandPalette.preserveInput": true,
  "workbench.enableExperiments": false,
  "workbench.sideBar.location": "left",
  "workbench.tree.expandMode": "singleClick",
  "workbench.startupEditor": "welcomePage",
  "workbench.editor.pinnedTabSizing": "compact",
  "workbench.editor.limit.enabled": true,
  "workbench.editor.limit.perEditorGroup": true,
  "workbench.editor.limit.value": 10,
  "workbench.editor.highlightModifiedTabs": true,
  "workbench.editor.enablePreviewFromQuickOpen": true,
  "workbench.editor.enablePreviewFromCodeNavigation": true,
  "workbench.editor.enablePreview": true,
  "workbench.editor.restoreViewState": true,
  "workbench.editor.closeOnFileDelete": true,
  "workbench.editor.tabSizing": "shrink",
  "workbench.editor.decorations.badges": true,
  "workbench.editor.decorations.colors": true,
  "workbench.settings.enableNaturalLanguageSearch": false,
  "workbench.iconTheme": "material-icon-theme",
  "workbench.colorTheme": "Monokai",
  "material-icon-theme.showWelcomeMessage": false,
  "diffEditor.ignoreTrimWhitespace": false,
  "diffEditor.codeLens": true,
  "vsintellicode.modify.editor.suggestSelection": "choseToUpdateConfiguration",
  "window.title": "${dirty} ${activeEditorMedium}${separator}${rootName}",
  "window.nativeTabs": true,
  "window.titleBarStyle": "custom",
  "window.restoreFullscreen": true,
  "window.restoreWindows": "all",
  "javascript.referencesCodeLens.enabled": true,
  "javascript.referencesCodeLens.showOnAllFunctions": true,
  "javascript.implementationsCodeLens.enabled": true,
  "javascript.implementationsCodeLens.showAllFunctions": true,
  "javascript.format.insertSpaceAfterSemicolonInForStatements": false,
  "javascript.suggest.autoImports": true,
  "javascript.suggest.includeAutomaticOptionalChainCompletions": true,
  "javascript.updateImportsOnFileMove.enabled": "always",
  "javascript.autoClosingTags": true,
  "javascript.validate.enable": false,
  "javascript.format.enable": false,
  "typescript.referencesCodeLens.enabled": true,
  "typescript.referencesCodeLens.showOnAllFunctions": true,
  "typescript.implementationsCodeLens.enabled": true,
  "typescript.implementationsCodeLens.showAllFunctions": true,
  "typescript.format.insertSpaceAfterSemicolonInForStatements": false,
  "typescript.suggest.autoImports": true,
  "typescript.suggest.paths": true,
  "typescript.suggest.includeAutomaticOptionalChainCompletions": true,
  "typescript.suggestionActions.enabled": true,
  "typescript.updateImportsOnFileMove.enabled": "always",
  "typescript.autoClosingTags": true,
  "typescript.validate.enable": true,
  "typescript.format.enable": false,
  "typescript.tsserver.useSeparateSyntaxServer": true,
  "typescript.tsserver.experimental.enableProjectDiagnostics": false,
  "typescript.tsserver.maxTsServerMemory": 2048,
  "typescript.tsserver.log": "off",
  "typescript.disableAutomaticTypeAcquisition": false,
  "typescript.surveys.enabled": false,
  "js/ts.implicitProjectConfig.checkJs": true,
  "js/ts.implicitProjectConfig.experimentalDecorators": true,
  "js/ts.implicitProjectConfig.strictNullChecks": true,
  "js/ts.implicitProjectConfig.strictFunctionTypes": true,
  "css.validate": false,
  "less.validate": false,
  "scss.validate": false,
  "stylelint.enable": true,
  "python.jediEnabled": true,
  "python.linting.pylamaEnabled": true,
  "explorer.confirmDelete": false,
  "explorer.confirmDragAndDrop": false,
  "explorer.openEditors.visible": 10,
  "files.watcherExclude": {
    "**/.cache/**": true,
    "**/.webpack/**": true,
    "**/.idea/**": true,
    "**/node_modules/**": true,
    "**/build": true,
    "**/dist": true,
    "**/tmp": true
  },
  "files.exclude": {},
  "files.autoSave": "onWindowChange",
  "files.trimTrailingWhitespace": true,
  "files.insertFinalNewline": false,
  "files.associations": {
    "*.js": "javascriptreact",
    "*.ts": "typescript"
  },
  "files.saveConflictResolution": "askUser",
  "files.restoreUndoStack": true,
  "emmet.includeLanguages": {
    "vue-html": "html",
    "postcss": "css",
    "javascript": "javascriptreact",
    "markdown": "html"
  },
  "emmet.syntaxProfiles": {
    "postcss": "css"
  },
  "emmet.triggerExpansionOnTab": true,
  "emmet.showExpandedAbbreviation": "never",
  "emmet.showSuggestionsAsSnippets": true,
  "html.autoClosingTags": true,
  "git.openAfterClone": "prompt",
  "git.enableSmartCommit": false,
  "git.useCommitInputAsStashMessage": true,
  "git.autofetch": "all",
  "git.pruneOnFetch": true,
  "git.confirmSync": false,
  "git.rebaseWhenSync": true,
  "git.followTagsWhenSync": false,
  "git.pullTags": false,
  "git.supportCancellation": false,
  "git.checkoutType": ["local", "remote", "tags"],
  "scm.alwaysShowActions": true,
  "scm.alwaysShowRepositories": true,
  "scm.defaultViewMode": "tree",
  "scm.providerCountBadge": "auto",
  "scm.diffDecorationsGutterAction": "diff",
  "merge-conflict.codeLens.enabled": true,
  "search.exclude": {
    "**/.cache": true,
    "**/.git": true,
    "**/.out": true,
    "**/build": true,
    "**/dist": true,
    "**/tmp": true,
    "**/node_modules": true,
    "**/bower_packages": true,
    "**/bower_components": true
  },
  "search.searchOnType": false,
  "search.collapseResults": "alwaysCollapse",
  "search.mode": "reuseEditor",
  "search.experimental.searchInOpenEditors": true,
  "task.autoDetect": "on",
  "task.quickOpen.detail": true,
  "npm.autoDetect": "on",
  "npm.packageManager": "auto",
  "npm.fetchOnlinePackageInfo": true,
  "go.gopath": "~/go",
  "go.toolsGopath": "~/go/bin",
  "go.formatTool": "goimports",
  "go.useLanguageServer": true,
  "go.languageServerFlags": ["serve", "-rpc.trace", "--debug=localhost:6060"],
  "reason_language_server.location": "/usr/local/bin/reason-language-server",
  "rust-analyzer.updates.askBeforeDownload": true,
  "rust-analyzer.lens.methodReferences": true,
  "docker.images.label": "Tag",
  "python.languageServer": "Pylance",
  "python.analysis.completeFunctionParens": true,
  "python.analysis.typeCheckingMode": "strict",
  "jupyter.enableCellCodeLens": true,
  "notebook.breadcrumbs.showCodeCells": true,
  "notebook.outline.showCodeCells": true,
  "java.implementationsCodeLens.enabled": true,
  "java.project.importHint": true,
  "cSpell.enabled": false,
  "settingsSync.keybindingsPerPlatform": false,
  "timeline.excludeSources": [],
  "stylelint.packageManager": "yarn",
  "stylelint.validate": [
    "css",
    "html",
    "javascriptreact",
    "less",
    "markdown",
    "postcss",
    "sass",
    "scss",
    "source.css.styled",
    "source.markdown.math",
    "styled-css",
    "sugarss",
    "svelte",
    "typescript",
    "typescriptreact",
    "vue",
    "vue-html",
    "vue-postcss",
    "xml",
    "xsl"
  ]
}
```
</details>

**Comment by bpasero** [comment]
This seems relevant:

```
2021-02-22 19:49:37.303 Electron[66422:1103275] *** Assertion failure in -[NSWindowStackController _enterWindow:intoFullScreenWithWindow:], /AppleInternal/BuildRoot/Library/Caches/com.apple.xbs/Sources/AppKit/AppKit-1894.60.100/WindowTabs/NSWindowStackController.m:999
[0222/194937.312789:WARNING:process_memory_mac.cc(93)] mach_vm_read(0x7ffee79f5000, 0x2000): (os/kern) invalid address (1)
[0222/194937.423040:WARNING:crash_report_exception_handler.cc(240)] UniversalExceptionRaise: (os/kern) failure (5)
[66431:0222/194937.427509:ERROR:child_thread_impl.cc(222)] Invalid PlatformChannel receive right
```

**Comment by bpasero** [comment]
@bpasero btw i was not able to reproduce even with your settings. can you distill the exact steps that leads to this possibly by starting from a fresh user data dir (using `code-insiders --user-data-dir <some empty folder>`) and going step by step?

**Comment by bpasero** [comment]
Parts of the crash remind me of https://github.com/microsoft/vscode/issues/65115 ...

**Comment by dalisoft** [comment]
@bpasero Try with fresh install?

**Comment by bpasero** [comment]
No it can be done from the existing install.

**Comment by dalisoft** [comment]
I mean i try after removing user folder? Are you able to reproduce bug? Maybe it's macOS issue (my macOS updates issue)

**Comment by bpasero** [comment]
I cannot reproduce the issue that is why I was asking if you can start from a fresh user data dir as outlined by the command line otion.

**Comment by dalisoft** [comment]
I tried with empty config

```bash
❯ code --verbose
(electron) Sending uncompressed crash reports is deprecated and will be removed in a future version of Electron. Set { compress: true } to opt-in to the new behavior. Crash reports will be uploaded gzipped, which most crash reporting servers support.
[70549:0222/210828.796040:WARNING:dns_config_service_posix.cc(185)] dns_config has unhandled options!
[main 2021-02-22T16:08:28.902Z] Starting VS Code
[main 2021-02-22T16:08:28.902Z] from: /Applications/Visual Studio Code.app/Contents/Resources/app
[main 2021-02-22T16:08:28.902Z] args: {
  _: [],
  diff: false,
  add: false,
  goto: false,
  'new-window': false,
  'reuse-window': false,
  wait: false,
  help: false,
  'list-extensions': false,
  'show-versions': false,
  version: false,
  verbose: true,
  status: false,
  'prof-startup': false,
  'prof-v8-extensions': false,
  'disable-extensions': false,
  'disable-gpu': false,
  telemetry: false,
  debugRenderer: false,
  logExtensionHostCommunication: false,
  'skip-release-notes': false,
  'disable-telemetry': false,
  'disable-updates': false,
  'disable-crash-reporter': false,
  'skip-add-to-recently-opened': false,
  'unity-launch': false,
  'open-url': false,
  'file-write': false,
  'file-chmod': false,
  'driver-verbose': false,
  force: false,
  'do-not-sync': false,
  trace: false,
  'force-user-env': false,
  'force-disable-user-env': false,
  'open-devtools': false,
  __sandbox: false,
  'no-proxy-server': false,
  nolazy: false,
  'force-renderer-accessibility': false,
  'ignore-certificate-errors': false,
  'allow-insecure-localhost': false,
  logsPath: '/Users/dalisoft/Library/Application Support/Code/logs/20210222T210828'
}
[main 2021-02-22T16:08:28.907Z] Resolving machine identifier...
[main 2021-02-22T16:08:28.908Z] Resolved machine identifier: 6a3cf7b0ca7fed5ab37c6479feaa0e062f0b729f22f6e48d85857561a02e792b
[main 2021-02-22T16:08:28.908Z] Main->SharedProcess#connect
[main 2021-02-22T16:08:28.935Z] lifecycle (main): phase changed (value: 2)
[main 2021-02-22T16:08:28.935Z] windowsManager#open
[main 2021-02-22T16:08:28.936Z] windowsManager#open pathsToOpen [
  {
    workspace: { id: '3616f9c71988d4ad5dd50fb21055f3cc', uri: [v] },
    exists: true
  },
  {
    workspace: { id: '98cc6cfb5c41e53462ffbfe01a2e3d2a', uri: [v] },
    exists: true
  },
  {
    workspace: { id: '05231ba37ee0454d7d959cd064ae41d8', uri: [v] },
    exists: true
  },
  {
    workspace: { id: '7278d4ec3a8e518ce26d9f90330c8d5f', uri: [v] },
    exists: true
  },
  {
    workspace: { id: '98cc6cfb5c41e53462ffbfe01a2e3d2a', uri: [v] },
    exists: true
  }
]
[main 2021-02-22T16:08:28.943Z] window#validateWindowState: validating window state on 1 display(s) { mode: 1, x: 768, y: 185, width: 1509, height: 919 }
[main 2021-02-22T16:08:28.943Z] window#validateWindowState: 1 monitor working area { x: 0, y: 23, width: 2560, height: 1353 }
[main 2021-02-22T16:08:28.943Z] window#ctor: using window state { mode: 1, x: 768, y: 185, width: 1509, height: 919 }
[70553:0222/210828.988043:WARNING:dns_config_service_posix.cc(185)] dns_config has unhandled options!
[main 2021-02-22T16:08:29.016Z] window#validateWindowState: validating window state on 1 display(s) {
  mode: 3,
  display: 69967544,
  width: 1509,
  height: 919,
  x: 768,
  y: 185
}
[main 2021-02-22T16:08:29.016Z] window#validateWindowState: 1 monitor working area { x: 0, y: 23, width: 2560, height: 1353 }
[main 2021-02-22T16:08:29.016Z] window#ctor: using window state {
  mode: 3,
  display: 69967544,
  width: 1509,
  height: 919,
  x: 768,
  y: 185
}
[main 2021-02-22T16:08:29.153Z] window#validateWindowState: validating window state on 1 display(s) { mode: 1, x: 773, y: 457, width: 1509, height: 919 }
[main 2021-02-22T16:08:29.153Z] window#validateWindowState: 1 monitor working area { x: 0, y: 23, width: 2560, height: 1353 }
[main 2021-02-22T16:08:29.153Z] window#ctor: using window state { mode: 1, x: 773, y: 457, width: 1509, height: 919 }
[main 2021-02-22T16:08:29.187Z] window#validateWindowState: validating window state on 1 display(s) {
  mode: 1,
  display: 69967544,
  width: 1024,
  height: 768,
  x: 828,
  y: 396
}
[main 2021-02-22T16:08:29.187Z] window#validateWindowState: 1 monitor working area { x: 0, y: 23, width: 2560, height: 1353 }
[main 2021-02-22T16:08:29.187Z] window#ctor: using window state {
  mode: 1,
  display: 69967544,
  width: 1024,
  height: 768,
  x: 828,
  y: 396
}
2021-02-22 21:08:29.199 Electron[70549:1180338] *** Assertion failure in -[NSWindowStackController _enterWindow:intoFullScreenWithWindow:], /AppleInternal/BuildRoot/Library/Caches/com.apple.xbs/Sources/AppKit/AppKit-1894.60.100/WindowTabs/NSWindowStackController.m:999
[0222/210829.211911:WARNING:process_memory_mac.cc(93)] mach_vm_read(0x7ffee4324000, 0x2000): (os/kern) invalid address (1)
[0222/210829.331325:WARNING:crash_report_exception_handler.cc(240)] UniversalExceptionRaise: (os/kern) failure (5)
[70557:0222/210829.337006:ERROR:child_thread_impl.cc(222)] Invalid PlatformChannel receive right
~ 
❯ 
```

Issue happened only after setting `window.restoreFullscreen` to `true`, else works, but on restore, all tabs are merged into one

**Comment by deepak1556** [comment]
https://github.com/microsoft/vscode/issues/117220#issuecomment-783468156 indicates a crash, can you follow these steps https://github.com/microsoft/vscode/wiki/Native-Crash-Issues#creating-and-symbolicating-local-crash-reports with insiders version and attach the crash logs, thanks!

**Comment by deepak1556** [comment]
@bpasero the user is on Catalina, it could also be a crash only on this OS version.

**Comment by bpasero** [comment]
Yeah good point.

**Comment by dalisoft** [comment]
@bpasero I can confirm latest stable release fixed this issue, but introduces new one. Thanks to all developers and team for hard work

**Comment by dalisoft** [comment]
Insiders version also updated, but bug still exists
